weave-js/src/components/PagePanelComponents/Home/Browse3/context.tsx

```diff
@@ -417,6 +417,7 @@ export const browse3ContextGen = (
         primaryDim
       )}&secondaryDim=${encodeURIComponent(secondaryDim)}`;
     },
+    refetchCalls: () => {},
```

Comment:
This does not belong in the Url context. I'll keep reading and suggest an alternative place